### DISCUSSION TITLE
unused codes in non-publication-identifier

### DISCUSSION THREAD

**Discussion by renestreich** [OP-TED/eForms-SDK#955] [Q&A] [answered]
Hi,

going through the codelist non-publication-identifier I noticed that there are two codes that are used in the schematrons, but seem to be unused in the fields.json:

not-dis-dat
not-pub-dat

Are the aforementioned codes subject to introduction into fields.json in the future or are these leftovers that could be removed?

Best regards
René Streich

**Comment by YvesJo** [comment]
Hi,
Thanks for reporting this. We'll clarify this and get back to you.
KR

**Reply by YvesJo** [reply, marked as answer]
Hi,
These two codes will be removed from the codelist in SDK 1.13.
KR,
